docker/build_activationtck.sh

```diff
@@ -56,14 +56,13 @@ which mvn
 mvn -version
 
 export ANT_OPTS="-DTS_HOME=$WORKSPACE -DJAVA_HOME=$JAVA_HOME -DJARPATH=$WORKSPACE"
-ant -f release.xml clean mvn
-
 export JAVA_HOME=$JDK11_HOME
```

Comment:
This is outside the changes in this PR but noticing that JAVA_HOME is set as JDK11_HOME here. Previously JDK11_HOME was set as JDK8 by default and did not affect. I presume for Jakarta 9.1, the jdk11 compatability is required only for tck run and not required for building. If so we can remove this line and the next ?

Reply:
I agree with removing these lines - basically making JDK 11 (or newer) default and explicitly setting ie JDK8 when required for run. I did not make this change intentionally as I have no idea how can I easily test it (effectively meaning either leaving it up to someone else or for another PR :-) )